.travis.yml

```diff
@@ -49,17 +49,20 @@ matrix:
 
     # Job 7 ... ARMv7 cross compile
     - name: Linux ARMv7 Qemu GCC 7
-      env: MATRIX_EVAL="CC=gcc-7 && CXX=g++-7 && CMAKE_ARG=-DCMAKE_TOOLCHAIN_FILE=../cmake/Toolchains/arm-linux-gnueabihf.cmake"
+      env: MATRIX_EVAL="CC=gcc-7 && CXX=g++-7 && CMAKE_ARG=\"-DBUILD_PIC=ON -DCMAKE_TOOLCHAIN_FILE=../cmake/Toolchains/arm-linux-gnueabihf.cmake\" && QEMU_ARCH=arm"
       addons: {apt: {sources: "ubuntu-toolchain-r-test", packages: [*common_packages, g++-arm-linux-gnueabihf, qemu-user]}}
 
     # Job 8 ... ARMv8 (aarch64) cross compile
     - name: Linux ARMv8 (aarch64) Qemu GCC 7
-      env: MATRIX_EVAL="CC=gcc-7 && CXX=g++-7 && CMAKE_ARG=-DCMAKE_TOOLCHAIN_FILE=../cmake/Toolchains/aarch64-linux-gnu.cmake"
+      env: MATRIX_EVAL="CC=gcc-7 && CXX=g++-7 && CMAKE_ARG=\"-DBUILD_PIC=ON -DCMAKE_TOOLCHAIN_FILE=../cmake/Toolchains/aarch64-linux-gnu.cmake\""
+           TOOLCHAIN=LINARO
+           TARGET=aarch64-linux-gnu
+           QEMU_ARCH=aarch64
```

Comment:
@noc0lour So we should do this:
https://github.com/google/cpu_features/blob/6d0767e0d9269919452dfa20a326cb4a6529171a/.travis.yml#L56

here as well? For some reason inline code links don't work.

Reply:
These env variables were a really naive approach. The cross-compile etc. build needs more attention.

@noc0lour thanks for your suggestions.